api/src/pcapi/core/finance/api.py

```diff
@@ -532,15 +532,22 @@ def _generate_business_units_file() -> pathlib.Path:
     )
     row_formatter = lambda row: (
         human_ids.humanize(row.venue_id),
-        row.business_unit_siret,
-        row.business_unit_name,
-        row.venue_name,
+        _clean_trailing_newline(row.business_unit_siret),
+        _clean_trailing_newline(row.business_unit_name),
+        _clean_trailing_newline(row.venue_name),
         row.iban,
         row.bic,
     )
     return _write_csv("business_units", header, rows=query, row_formatter=row_formatter)
 
 
+def _clean_trailing_newline(row: str) -> str:
+    """remove trailing new line if string --> intended for BU, venue and offerer name"""
+    if isinstance(row, str):
+        return row.rstrip("\n")
+    return row
```

Comment:
(détail / question de goût)

Je propose de traiter le cas particulier en premier. Ça permet de ne pas avoir à indenter la logique principale. (Même si ici la logique tient sur une ligne, ça pourrait évoluer.)

```suggestion
    if not isinstance(value, str):
        return value
    return value.rstrip("\n")
```